metadata.json

```diff
@@ -12,8 +12,8 @@
     {"name":"puppet","version_requirement":">= 3.0.0 < 7.0.0"}
   ],
   "dependencies": [
-    {"name":"puppetlabs/sshkeys_core","version_requirement":">= 1.0.1 <2.0.0"},
-    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 6.0.0"}
+    {"name":"puppetlabs/sshkeys_core","version_requirement":">= 1.0.1 <4.0.0"},
+    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 8.0.0"}
```

Comment:
stdlib 7 doesnt exist and might not be compatible with this version of the code.